tests/acceptance/pageObjects/personalPage.js

```diff
@@ -369,12 +353,12 @@ module.exports = {
       selector: '//span[@class="oc-breadcrumb-drop-label-text" and text()=%s]',
       locateStrategy: 'xpath'
     },
-    resourceBreadcrumb: {
+    breadcrumbMobileReferencedToOpenSidebarButton: {
       selector:
-        '//nav[@id="files-breadcrumb"]//*[(self::a or self::span or self::button) and contains(text(),%s)]',
+        '//button[@aria-label="Open sidebar to view details"]/ancestor::div//span[@class="oc-breadcrumb-drop-label-text" and text()=%s]',
       locateStrategy: 'xpath'
     },
```

Comment:
This new selector has been added, to find out if the breadcrumb is actually visible. This is not a good approach though. I could not find any other way. I have commented in the issue itself. https://github.com/owncloud/web/issues/5790#issuecomment-923878811 . I would appreciate any other suggestions